src/CalculatorUnitTests/DateCalculatorUnitTests.cpp

```diff
@@ -955,7 +956,7 @@ TEST_METHOD(DateCalcViewModelAddSubtractResultAutomationNameTest)
 
 TEST_METHOD(JaEraTransitionAddition)
 {
-    auto viewModel = make_unique<DateCalculationEngine>(CalendarIdentifiers::Japanese);
+    auto viewModel = ref new DateCalculationEngine(CalendarIdentifiers::Japanese);
```

Comment:
nit: while you're here, let's name the variable "engine" instead of "viewModel" in all the JaEra* tests